src/data/Team.json

```diff
@@ -0,0 +1,69 @@
+[
+  {
+    "title": "Ton",
+    "position": "Senior Developer",
```

Comment:
May I ask you to change my job position to `Software Applications Architect`  please.